Goal: kolab.globals

- Use syncrepl LDAP backend (OpenLDAP >= 2.3)
- Use syslog

Author: Mathieu Parent <math.parent@gmail.com>

Upstream status: Already included, will be in 2.2.1 (kolab/issue2446)

Index: b/kolab.globals.in
===================================================================
--- a/kolab.globals.in
+++ b/kolab.globals.in
@@ -7,7 +7,7 @@
 cyrus-pop3 : FALSE
 cyrus-pop3s : TRUE
 cyrus-sieve : TRUE
-directory_mode : slurpd
+directory_mode : syncrepl
 gyard_deletion_period : 10080
 ldap_uri : ldap://127.0.0.1:389
 log_level : 2
@@ -48,4 +48,4 @@
 bind_any : TRUE
 bind_addr : 0.0.0.0
 calendar_id : calendar
-syslog_facility : user
+syslog_facility : local5
